internal/storagetools/check.go

```diff
@@ -0,0 +1,61 @@
+package storagetools
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+
+	"github.com/wal-g/tracelog"
+)
+
+func HandleCheckRead(folder storage.Folder, filenames []string) {
+	objs, _, err := folder.ListFolder()
```

Comment:
This will only list the objects of the root folder. I guess that it will not work if the filename is `folder1/subfolder/file`.

Reply:
well.. yes.
It's not required for my case, as all files are supposed to be in one folder.
```
WALG_S3_PREFIX: "s3://se-test/subfolder"
```
may we implement subfolder checks later ?

Reply:
fixed